src/mame/machine/ibm6580_kbd.cpp

```diff
@@ -89,69 +89,69 @@ INPUT_PORTS_START( dw_keyboard )
 
 	PORT_START("COL.3")
 	PORT_BIT( 0x01, IP_ACTIVE_HIGH, IPT_KEYBOARD ) PORT_NAME("/ ?") PORT_CODE(KEYCODE_SLASH) PORT_CHAR('/') PORT_CHAR('?')
-	PORT_BIT( 0x02, IP_ACTIVE_HIGH, IPT_KEYBOARD ) PORT_NAME("3 2") PORT_CODE(KEYCODE_BACKSLASH)
+	PORT_BIT( 0x02, IP_ACTIVE_HIGH, IPT_KEYBOARD ) PORT_NAME("3 2") PORT_CODE(KEYCODE_BACKSLASH) PORT_CHAR('\\') PORT_CHAR('|')
```

Comment:
Is this for superscript 3 and 2?  If it is, the `PORT_CHAR` should be 0xb2 and 0xb3, not something unrelated.